dashboard/app/models/sections/section.rb

```diff
@@ -584,7 +584,7 @@ def update_code_review_expiration(enable_code_review)
   end
   before_validation :strip_emoji_from_name
 
-  public def add_instructor(email)
+  public def add_instructor(email, current_user)
```

Comment:
I think `invited_by` is a better descriptor of what this parameter is actually for.

```suggestion
  public def add_instructor(email, invited_by)
```

Switching it up to keyword parameters instead of positional parameters would make the purpose of passing `current_user` more clear in the calling sites, at the cost of slightly longer code. I'll leave it up to you whether that's worth it.

```suggestion
  public def add_instructor(email:, invited_by:)
```